extras/Makefile.am

```diff
@@ -39,15 +39,17 @@ scripts_SCRIPTS += control-mem.sh
 endif
 endif
 
+CLEANFILES = command-completion/Makefile command-completion/README
 EXTRA_DIST = glusterfs-logrotate gluster-rsyslog-7.2.conf gluster-rsyslog-5.8.conf \
 	logger.conf.example glusterfs-georep-logrotate group-virt.example \
 	group-metadata-cache group-gluster-block group-nl-cache \
 	group-db-workload group-samba specgen.scm glusterfs-mode.el glusterfs.vim \
 	migrate-unify-to-distribute.sh backend-xattr-sanitize.sh \
 	backend-cleanup.sh disk_usage_sync.sh clear_xattrs.sh \
 	glusterd-sysconfig glusterd.vol post-upgrade-script-for-quota.sh \
-	pre-upgrade-script-for-quota.sh command-completion/gluster.bash \
-	command-completion/Makefile command-completion/README \
+	pre-upgrade-script-for-quota.sh \
+	command-completion/gluster.bash command-completion/README.in \
+	command-completion/Makefile.am command-completion/Makefile.in \
```

Comment:
Not sure if this is necessary

Reply:
As I understand it, the Makefile is not run during install, the files are not taken into account as dist files. Please see my comment on the Readme.in file.

Reply:
Understood.

Reply:
did you consider adding `command-completion` to `SUBDIRS`? That might be cleaner and gives you the option to do include a `install-data-local` target so that `make install` does it all automatically, no need to manually copy things around in the spec file.

Reply:
I just pushed a commit to include `command-completion` in the `SUBDIRS` for extras, so that the bash completion is installed by default. It is simpler overall, indeed.

During an installation from source, the cli seems to always be installed, so I guess the bash-completion script can be installed by default too. It was not the case before this PR and this commit: you can include the commit if you think it is ok.